code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -172,11 +172,12 @@
 
 	stacktypes = list(
 		/obj/item/stack/sheet/metal = 50,
+		/obj/item/stack/sheet/plasteel = 50,
 		/obj/item/stack/sheet/glass = 50,
 		/obj/item/stack/sheet/rglass = 50,
-		/obj/item/stack/cable_coil/cyborg = 50,
-		/obj/item/stack/rods = 15,
-		/obj/item/stack/tile/plasteel = 15
+		/obj/item/stack/cable_coil/cyborg = 30,

```

Comment:
@Regen1 you don't need to be an ass about it, it was an accident by not verifying against the repo code